agent/main.go

```diff
@@ -71,6 +71,11 @@ func main() {
 		log.G(shimCtx).WithError(err).Fatal("failed to set shim as subreaper")
 	}
 
+	_, err := os.Create("/verify-rootfs")
```

Comment:
The agent will likely have a writable view of the root filesystem (at least in the image built from our image-builder) as it runs in a writable portion of overlayfs.  I don't think we need to verify whether it's writable from within the agent, as that's already part of the system that runs in a potentially-hostile environment.  I think it should be sufficient to ensure that we attach the drive as read-only when we configure the VM.

Reply:
Discussed offline: To automatically and continuously verify the protection, we can put a malicious agent, rather than let the production agent do things that should fail.

Reply:
Removed on the last revision.